Kernel/DoubleBuffer.cpp

```diff
@@ -77,4 +77,24 @@ ssize_t DoubleBuffer::read(UserOrKernelBuffer& data, size_t size)
     return (ssize_t)nread;
 }
 
+ssize_t DoubleBuffer::peek(UserOrKernelBuffer& data, size_t size)
+{
+    if (!size || m_storage.is_null())
+        return 0;
+    VERIFY(size > 0);
```

Comment:
This VERIFY is essentially useless, as the if directly before it ensures that it will never be hit (because size_t is unsigned)

Reply:
Yeah, the original write() above does the same thing, I'll fix it for both.